uber/templates/regform.html

```diff
@@ -52,15 +52,15 @@
 
     {% if not attendee.is_new %}
         // This removes any badge levels lower than the attendee has purchased already
-        while (_(BADGE_TYPES.options).size() && BADGE_TYPES.options[0].extra < {{ attendee.amount_extra }}) {
```

Comment:
I need to unconvert some of these - I added the hidden `amount_extra` field rather late in the refactor, and there's definitely cases for relying on it to use the same integer comparison we used to.